### PR TITLE
BZ2055540: Correct ConfigMap example indentation

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
For 4.6+

https://bugzilla.redhat.com/show_bug.cgi?id=2055540

Requires ack from @evakhoni

### CODE REVIEW

**Comment by opayne1** [comment]
Updated: preview link: https://deploy-preview-42340--osdocs.netlify.app/openshift-enterprise/latest/rest_api/config_apis/proxy-config-openshift-io-v1.html

File: rest_api/config_apis/proxy-config-openshift-io-v1.adoc [hunk 1]
```diff
@@ -63,40 +63,65 @@ Type::
...
- The namespace for the ConfigMap referenced by trustedCA is "openshift-config". Here is an example ConfigMap (in yaml): 
- apiVersion: v1 kind: ConfigMap metadata:  name: user-ca-bundle  namespace: openshift-config  data:    ca-bundle.crt: \|      -----BEGIN CERTIFICATE-----      Custom CA certificate bundle.      -----END CERTIFICATE-----
+| `trustedCA` is a reference to a ConfigMap containing a CA certificate bundle. The trustedCA field should only be consumed by a proxy validator. The validator is responsible for reading the certificate bundle from the required key `ca-bundle.crt`, merging it with the system default trust bundle, and writing the merged trust bundle to a ConfigMap named `trusted-ca-bundle` in the `openshift-config-managed` namespace. Clients that expect to make proxy connections must use the trusted-ca-bundle for all HTTPS requests to the proxy, and may use the trusted-ca-bundle for non-proxy HTTPS requests as well.
```
Comment (opayne1) [inline rest_api/config_apis/proxy-config-openshift-io-v1.adoc:88]:
```suggestion
| `trustedCA` is a reference to a ConfigMap containing a CA certificate bundle. The trustedCA field should only be consumed by a proxy validator. The validator is responsible for reading the certificate bundle from the required key `ca-bundle.crt`, merging it with the system default trust bundle, and writing the merged trust bundle to a ConfigMap named `trusted-ca-bundle` in the `openshift-config-managed` namespace. Clients that expect to make proxy connections must use the `trusted-ca-bundle` for all HTTPS requests to the proxy, and may use the `trusted-ca-bundle` for non-proxy HTTPS requests as well.
```

File: rest_api/config_apis/proxy-config-openshift-io-v1.adoc [hunk 2]
```diff
@@ -63,40 +63,65 @@ Type::
...
- The namespace for the ConfigMap referenced by trustedCA is "openshift-config". Here is an example ConfigMap (in yaml): 
- apiVersion: v1 kind: ConfigMap metadata:  name: user-ca-bundle  namespace: openshift-config  data:    ca-bundle.crt: |      -----BEGIN CERTIFICATE-----      Custom CA certificate bundle.      -----END CERTIFICATE-----
+`trustedCA` is a reference to a ConfigMap containing a CA certificate bundle. The trustedCA field should only be consumed by a proxy validator. The validator is responsible for reading the certificate bundle from the required key `ca-bundle.crt`, merging it with the system default trust bundle, and writing the merged trust bundle to a ConfigMap named `trusted-ca-bundle` in the `openshift-config-managed` namespace. Clients that expect to make proxy connections must use the trusted-ca-bundle for all HTTPS requests to the proxy, and may use the trusted-ca-bundle for non-proxy HTTPS requests as well.
```
Comment (opayne1) [inline rest_api/config_apis/proxy-config-openshift-io-v1.adoc:109]:
```suggestion
`trustedCA` is a reference to a ConfigMap containing a CA certificate bundle. The trustedCA field should only be consumed by a proxy validator. The validator is responsible for reading the certificate bundle from the required key `ca-bundle.crt`, merging it with the system default trust bundle, and writing the merged trust bundle to a ConfigMap named `trusted-ca-bundle` in the `openshift-config-managed` namespace. Clients that expect to make proxy connections must use the `trusted-ca-bundle` for all HTTPS requests to the proxy, and may use the `trusted-ca-bundle` for non-proxy HTTPS requests as well.
```

**Review by opayne1 — COMMENTED** [review]
Just a few small things :)

**Comment by kalexand-rh** [comment]
This PR only modifies a REST API file, so any changes to it would be overwritten the next time the API docs are generated. Closing.

**Pull request closed without merging**